contracts/scripts/native_solc_compile_all_ccip

```diff
@@ -32,17 +33,14 @@ compileContract () {
       ;;
   esac
 
-  solc --overwrite --optimize --optimize-runs $optimize_runs --metadata-hash none \
+  solc --overwrite --via-ir --optimize --optimize-runs $optimize_runs --metadata-hash none \
       -o "$ROOT"/contracts/solc/v$SOLC_VERSION/"$contract" \
       --abi --bin --allow-paths "$ROOT"/contracts/src/v0.8 \
       --bin-runtime --hashes --metadata --metadata-literal --combined-json abi,hashes,metadata,srcmap,srcmap-runtime \
       --evm-version paris \
       "$ROOT"/contracts/src/v0.8/"$1"
 }
 
-
-# Solc produces and overwrites intermediary contracts.
```

Comment:
This comment isn't relevant as we compile to a separate dir per contract, not a shared folder